src/code/views/select-interactive-state-dialog-view.tsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react'
+
+import tr  from '../utils/translate'
+import ModalDialogView from './modal-dialog-view'
+import { IInteractiveStateProps } from '@concord-consortium/lara-interactive-api'
+
+interface SelectInteractiveStateVersionProps {
+  version: IInteractiveStateProps<{}>
+  showingOverlay: boolean
+  onSelect?: SelectInteractiveStateCallback
+  onPreview: (showing: boolean) => void
+  close?: () => void
+}
+
+interface SelectInteractiveStateVersionState {
```

Comment:
Since `SelectInteractiveStateVersionProps` and `SelectInteractiveStateVersionState` are only used internally, you could use `IProps` and `IState` which would improve readability below.